packages/online-editor/src/env/EnvJson.ts

```diff
@@ -16,6 +16,7 @@
 
 import { AuthProvider } from "../authProviders/AuthProvidersApi";
 import { AcceleratorConfig } from "../accelerators/AcceleratorsApi";
+import { EditorConfig as EditorConfig } from "../envelopeLocator/EditorEnvelopeLocatorApi";
```

Comment:
```suggestion
import { EditorConfig } from "../envelopeLocator/EditorEnvelopeLocatorApi";
```

That's VS Code's fault, I know :P